CHANGELOG.md

```diff
@@ -2,6 +2,9 @@ v1.5.5
 
 Frontend Changes: 
 
+- Added WhatsApp, Viber, and VK to the Interactions by Channel chart list
```

Comment:
The changelog modification should create a new version `v1.5.6` with the modifications for this version. The added lines should not be in `v1.5.5`. @alviriseup

Reply:
Noted, and updated the CHANGELOG.md accordingly. @paulobernardoaf